docs/json_rpc_api.mediawiki

```diff
@@ -305,6 +428,47 @@ the method name for further details such as parameter and return information.
 
 ----
 
+{|
+!Method
+|createrawsstx
```

Comment:
Missing the `====createrawsstx====` header so that the link in the overview table work, it shows up in the auto generated ToC, and it's consistent with the other entries.